modules/graceful-shutdown.adoc

```diff
@@ -37,28 +37,27 @@ $ oc -n openshift-kube-apiserver-operator get secret kube-apiserver-to-kubelet-s
 ----
 <1> To ensure that the cluster can restart gracefully, plan to restart it on or before the specified date. As the cluster restarts, the process might require you to manually approve the pending certificate signing requests (CSRs) to recover kubelet certificates.
 
-. Shut down all of the nodes in the cluster. You can do this from your cloud provider's web console, or you can use the below commands:
-
-.. Obtain the list of nodes:
+. Shut down all of the nodes in the cluster. You can do this from your cloud provider's web console, or you can use the below command:
```

Comment:
```suggestion
. Shut down all of the nodes in the cluster. You can do this from your cloud provider's web console, or run the following loop:
```

Per the IBM style guide, you cannot use "below". Since this is a loop, my suggestion is above.

Reply:
Done